src/NorthwindWebsite/Core/ApplicationSettings/SerilogConfig.cs

```diff
@@ -8,6 +8,8 @@ public class SerilogConfig : IValidatable
     {
         public MinimumLevel MinimumLevel { get; set; }
 
+        public bool ActionsLogging { get; set; }
```

Comment:
Not really clear connection between type and var name: in this case I would represent `ActionsLogging` as string array that contains actions for logging. Let's name it like `ShouldLogControllersActions`. The word `Controllers` also specifies particular usage of this variable, because `Actions` it a little bit abstract)